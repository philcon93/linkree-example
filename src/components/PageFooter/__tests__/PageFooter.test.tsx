import { render } from '@testing-library/react';
import { PageFooter } from '../';

describe('PageFooter', () => {
    test('Displays correctly', () => {
        const { container } = render(
            <PageFooter />
        );

        expect(container).toMatchSnapshot();
    });
});
