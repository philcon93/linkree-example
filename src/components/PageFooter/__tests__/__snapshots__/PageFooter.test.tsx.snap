// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter Displays correctly 1`] = `
<div>
  <a
    class="chakra-link css-1s8lc44"
    href="https://linktr.ee/"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="Linktree logo"
      class="chakra-image__placeholder css-0"
    />
  </a>
</div>
`;
